src/table.mjs

```diff
@@ -73,7 +93,7 @@ async function* accumulateQuery(queryRequest) {
  * of sub-strings and params are the parameter values to be inserted between each
  * sub-string.
  */
- export function makeQueryTemplate(operations, source) {
+ export function makeQueryTemplate(source, operations) {
```

Comment:
note to self: revert this change because `makeQueryTemplate` is exported, so making this change would require a major version bump